src/tests/JIT/Regression/JitBlue/Runtime_34587/Runtime_34587.cs

```diff
@@ -713,7 +713,9 @@ static bool ValidateVectorT()
             if (Avx2IsSupported)
             {
                 succeeded &= VectorIsHardwareAccelerated;
-                succeeded &= VectorByteCount == 32;
+                // MaxVectorTBitWidth env variable can be used to change Vector<T> size.
+                // We can only assume it is at least 16 bytes.
+                succeeded &= VectorByteCount >= 16;
```

Comment:
I kept changes to the regression tests minimal -- this is handled more robustly in the CpuId tests.